lib/assembly/ecto_enums.ex

```diff
@@ -0,0 +1,2 @@
+import EctoEnum
+defenum(BuildStatusEnum, :status, [:incomplete, :ready, :built])
```

Comment:
Considering the work in converting between db to bottle status, we _could_ just do the bottle status here

Reply:
I would rather not. That couples our assembly service's data model to the messages we're passing around. They should be able to iterate independently of one another :+1: